### ISSUE TITLE
Some base.body.href.domain.tld false positives

### ISSUE THREAD

**Issue by sriccio** [martinschaible/rspamd-rules#79]
Hello Martin,

The rule base.body.href.domain.tld triggers false positives on:

```
anibis.help
```
```
tutti.help
```

Is there a way to whitelist them ?

Thanks and kind regards :)

**Comment by martinschaible** [comment]
The TLD "help" is blacklisted. We receive only spam from thes TLD.
I could whitelist the domain, but they are not working.
